server/start_celery.sh

```diff
@@ -1,3 +1,11 @@
-#!/bin/sh
+#!/bin/bash
 
-celery -A rematch.celery worker -l info
+if [[ -n "$IN_DOCKER" ]];
+then
+  echo "starting celeryin docker mode!"
+  while ! nc -z db 3306 ; do sleep 2; done
+  celery -A rematch.celery_docker worker -l info
+else
+  echo "starting celeryin debug mode!"
+  celery -A rematch.celery worker -l info
+fi
```

Comment:
every .sh is used to run things in background because of Docker's inability to run multiple scripts.